device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/pg_profile_lookup.ini

```diff
@@ -0,0 +1,23 @@
+#  PG lossless profiles.
```

Comment:
Should it be identical to SN4700? if so, why not link?

Reply:
I'm not sure that the values on this file will be the same as 4700 since we are using different gearboxes.
waiting for an answer from Matty.K about this issue.

Reply:
linked to SN4700